### PR TITLE
Use --depth 1 (shallow clone) for vagov-content repo.

### PR DESCRIPTION
**Pull request by ElijahLynn**
## Description
Speeds up the clone inside the VAEC network about 23 seconds.

image
image

### CODE REVIEW

File: script/install-repos.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 if [ ! -d ../vagov-content ]; then
-  git clone https://github.com/department-of-veterans-affairs/vagov-content.git ../vagov-content
+  git clone --depth 1 https://github.com/department-of-veterans-affairs/vagov-content.git ../vagov-content
```
Comment (ElijahLynn) [inline script/install-repos.sh]:
```suggestion
  git clone --single-branch --depth 1 https://github.com/department-of-veterans-affairs/vagov-content.git ../vagov-content
```

**Comment by ElijahLynn** [comment]
Single Pass Simple Benchmarks:
On server inside VAEC:
* time git clone https://github.com/department-of-veterans-affairs/vagov-content.git  38.76s
* time git clone --depth 1 https://github.com/department-of-veterans-affairs/vagov-content.git 14.0s
* time git clone --single-branch --depth 1 https://github.com/department-of-veterans-affairs/vagov-content.git 13.79s

**Comment by ElijahLynn** [comment]
Hrm, failing on npm security check, not my deal. 

image

**Comment by ElijahLynn** [comment]
@department-of-veterans-affairs/frontend-review-group Looking for a review on this, thanks.

**Review by cvalarida — APPROVED** [review]

**Pull request merged**